solar_installation/data/website_menu.xml

```diff
@@ -10,4 +10,10 @@
                 ('url', '=', '/default-main-menu'),
         ]).id"/>
     </record>
+    <function model="website.menu" name="unlink">
+        <value model="website.menu" eval="obj().search([
+            ('website_id', '=', ref('website.default_website')),
+            ('url', '=', '/contactus'),
+        ]).id"/>
+    </function>
```

Comment:
I think we should handle carefully existing dbs for now and avoid unlinking pages. Maybe only unpublish it in data? (and unlink in demo)